Makefile

```
@@ -77,14 +77,26 @@ clean: ## Delete build output
 
 ##@ Testing
 
-.PHONY: test
-test: generate ## Run all unit tests
+.PHONY: test-unit
+test-unit:
```

Most likely needs (try with `git clean -dfx`)

```suggestion
test-unit: generate
```

---

I was wondering about that. Is generate needed only for test-unit ?

---

Without it on clean checkout there is no code in `/application`...

---

Since generate runs whether it needs to or not, I decide to switch it back to a single task.